docs/user-guide/add-ai-assistant.md

```diff
@@ -0,0 +1,54 @@
+# Add Gen AI Assistant
+
+The KubeRocketCI portal allows you to integrate Large Language Models (LLMs for short) to help you by providing platform-related information and answering any questions. Assistant's configuration is performed via the Configuration tab, in the Gen AI section.
```

Comment:
The KubeRocketCI portal allows you to integrate Large Language Models (LLMs for short) it is not true